velox/benchmarks/tpch/CMakeLists.txt

```diff
@@ -37,6 +37,11 @@ target_link_libraries(
   ${FOLLY_BENCHMARK}
   fmt::fmt)
 
+if(VELOX_ENABLE_S3)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_S3FS")
```

Comment:
can we namespace this compilation flag with something like `VELOX_USE_S3FS` to prevent with possible clashes with other projects?

Reply:
We can use `add_definitions(-DVELOX_USE_S3FS)`

Reply:
Yes, done.